changelog/README.rst

```diff
@@ -0,0 +1,43 @@
+:orphan:
+
+Changelog
+=========
+
+This directory contains "news fragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next what's new page.
+
+Make sure to use full sentences with correct case and punctuation, and please
+try to use Sphinx intersphinx using backticks. The fragment should have a
+header line and an underline using ``------`` 
```

Comment:
I would be great to add a brief example here, e.g.,

```
my_new_feature option for np.my_favorite_function
-------------------------------------------------
The `my_new_feature` option is now available for `my_favorite_function`.
To use it, write ``np.my_favorite_function(..., my_new_feature=True)``.
```
with a note about how one backtick is used for the names of NumPy functions, and two backticks for code snippets (assuming I have that right).

Reply:
adding